client/src/controllers/BulkController.test.js

```diff
@@ -0,0 +1,58 @@
+import { Application } from '@hotwired/stimulus';
+import { BulkController } from './BulkController';
+
+describe('BulkController', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+    <div data-controller="w-bulk">
+      <input type="checkbox" data-w-bulk-target="all" data-action="w-bulk#toggleAll">
+      <div>
```

Comment:
```suggestion
      <input id="select-all" type="checkbox" data-w-bulk-target="all" data-action="w-bulk#toggleAll">
      <div id="checkboxes">
```

Add IDs, it's much easier to test DOM elements by finding by ID, also it makes it easier to change the controller identifier if we think of a better name later.

Reply:
yeah sure, IDs do make it better